template.fsf

```diff
@@ -248,10 +248,10 @@ set fmri(regstandard) "^STANDARD_TEMPLATE^"
 set fmri(regstandard_search) 90
 
 # Degrees of Freedom for registration to standard space
-set fmri(regstandard_dof) 12
+set fmri(regstandard_dof) ^DOF^
 
 # Do nonlinear registration from structural to standard space?
-set fmri(regstandard_nonlinear_yn) 0
+set fmri(regstandard_nonlinear_yn) ^FNIRT^
```

Comment:
What's funny is that even if I re-did this in python, this is still probably the easiest way to make these files.   I believe there is a deep dark hidden fsl tool that makes a feat file but it's pretty limited, as i recall.  ALL THINGS FOR GEAR V2.0!